.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,269 @@
+                } catch (e) {
+                    addError('Could not parse `community-css-themes.json`, invalid JSON. ' + e.message);
+                    return;
+                }
+                const theme = themes[themes.length - 1];
+                
+                let validPrKeys = ['name', 'author', 'repo', 'screenshot', 'modes'];
+                for (let key in validPrKeys) {
+                    if (!manifest.hasOwnProperty(key)) {
+                        addError(`Your PR does not have the required \`${key}\` property.`);
+                    }
+                }
+                for (let key in Object.keys(manifest)) {
+                    if (!validPrKeys.includes(key)) {
+                        addError(`Your manifest has the invalid \`${key}\` property.`);
+                    }
+                }
+
+                // Validate theme repo
+                let repoInfo = theme.repo.split('/');
+                if (repoInfo.length !== 2) {
+                    addError(`It seems like you made a typo in the repository field ${theme.repo}`);
+                    return;
+                }
+
+                let [owner, repo] = repoInfo;
+                console.log(`Repo info: ${owner}/${repo}`);
+
+                const author = context.payload.pull_request.user.login;
+                if (owner.toLowerCase() !== author.toLowerCase()) {
+                    try {
+                        const isInOrg = await github.rest.orgs.checkMembershipForUser({ org: owner, username: author });
+                        if (!isInOrg) {
+                            throw undefined;
+                        }
+                    } catch (e) {
+                        addError(`The newly added entry is not at the end, or you are submitting on someone else's behalf. The last plugin in the list is: \`${plugin.repo}\`.`);
+                    }
+                }
+                try {
+                    const repository = await github.rest.repos.get({ owner, repo });
+                    if (!repository.data.has_issues) {
+                        addWarning('Your repository does not have issues enabled. Users will not be able to report bugs and request features.');
+                    }
+                } catch (e) {
+                    addError(`It seems like you made a typo in the repository field ${theme.repo}`);
+                    return;
+                }
+
+                let manifest = {};
+
+                try {
+                    let manifestFile = await github.rest.repos.getContent({
+                        owner,
+                        repo,
+                        path: 'manifest.json',
+                    });
+
+                    manifest = JSON.parse(Buffer.from(manifestFile.data.content, 'base64').toString('utf-8'));
+                } catch (e) {
+                    addError(`You don't have a valid \`manifest.json\` at the root of your repo.`);
+                    return;
```

Comment:
This should not just return - we should deal with missing manifest with an if clause just like plugin reviews.
Doing it this way allows us to continue checking for more things like having a theme.css, so we can report as many problems in one go as possible.